server/models/nextlaunch.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+import "time"
+
+// NextLaunch struct for next launch revealing
+type NextLaunch struct {
+	ID        int       `json:"id" gorm:"primaryKey"`
+	Launched  bool      `json:"active"`
```

Comment:
why json name is active for Launched? they can both be `active`

Reply:
> why json name is active for Launched? they can both be `active`

You mean, it's better to change it> in order to separate between this struct and maintenance?

Reply:
no, I mean match the field name with its json tag

Reply:
> no, I mean match the field name with its json tag

aha, yes sure